app/src/main/java/ar/com/wolox/android/example/model/APIClient.java

```diff
@@ -0,0 +1,26 @@
+package ar.com.wolox.android.example.model;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+/**
+ * API Client - starts the session
+ */
+public final class APIClient {
```

Comment:
You can Inject an instance of retrofit from the presenter. I will show you how later if you want.